policy/policy.go

```diff
@@ -199,23 +197,21 @@ func (iamp Policy) IsAllowed(args Args) bool {
 	}
 
 	// Check all allow statements. If any one statement allows, return true.
-	if indexes, ok := iamp.actionStatementIndex[args.Action]; ok {
-		for _, index := range indexes {
-			statement := iamp.Statements[index]
-			if statement.Effect == Allow {
-				if statement.IsAllowed(args) {
+	if len(iamp.actionStatementIndex) > 0 {
+		if indexes, ok := iamp.actionStatementIndex[args.Action]; ok {
+			for _, index := range indexes {
+				statement := iamp.Statements[index]
+				if statement.Effect == Allow && statement.IsAllowed(args) {
 					return true
 				}
 			}
+			return false
```

Comment:
We need to not reintroduce the bug fixed in https://github.com/minio/pkg/pull/173